kernel/src/mm/ptguards.rs

```diff
@@ -16,6 +16,7 @@ use crate::types::{PAGE_SIZE, PAGE_SIZE_2M};
 
 use crate::utils::MemoryRegion;
 
+/// Guard for a per-CPU page mapping to ensure adequate cleanup if drop.
```

Comment:
"if drop" sounds a bit strange. Should be "when dropped":

```suggestion
/// Guard used to unmap a per-CPU mapping when dropped.
```